src/apis.rs

```diff
@@ -1,12 +1,11 @@
 /// Checks if, at some point in the protocol, the peer needs to verify the other peer static public
 /// key and if the peer needs to provide a proof for its static public key.
-
 use std::error;
 use std::fmt;
 
-use asymmetric::DH_SIZE;
-use config::Config;
-use disco::{DiscoReadError, DiscoWriteError, HandshakeState, KEY_SIZE};
+use crate::asymmetric::DH_SIZE;
+use crate::config::Config;
+use crate::disco::{DiscoReadError, DiscoWriteError, HandshakeState, KEY_SIZE};
```

Comment:
why adding crate here?

Reply:
it's how you reference the current crate in the 2018 edition. In the 2015 edition it was implicit.

- [0] https://doc.rust-lang.org/nightly/edition-guide/rust-2018/module-system/path-clarity.html

Reply:
oh right!